### PR TITLE
Support for humidity sensor

### PR DESCRIPTION
**Pull request by Adi-MMandroid**
### Tested on:
Device :-

### CODE REVIEW

**Comment by Adi-MMandroid** [comment]
@shubhamsinghmutualmobile not able to get this sensor to work on my device, isAvailable is false, could you get someone to test this please?

File: composesensors/src/main/java/com/mutualmobile/composesensors/RelativeHumiditySensorState.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        return 243.12 * (h / (17.62 - h))
+    }
+
+    fun getAbsHumidity(relativeHumidity: Float, actualTemp: Float): Double = 216.7 *
+        (
+            relativeHumidity / 100.0 * 6.112 * exp(
+                17.62 * actualTemp /
+                    (243.12 + actualTemp)
+            ) / (273.15 + actualTemp)
+            )
```
Comment (shubhamsinghmutualmobile) [inline composesensors/src/main/java/com/mutualmobile/composesensors/RelativeHumiditySensorState.kt:58]: Let's add the reference link(s) for these formulae in these functions' documentation, please.

**Comment by shubhamsinghmutualmobile** [comment]
@Adi-MMandroid We can remove the `[WIP]` prefix from this PR's title now 👍

File: composesensors/src/main/java/com/mutualmobile/composesensors/RelativeHumiditySensorState.kt [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    override fun toString(): String {
+        return "HumiditySensorState(relativeHumidity=$relativeHumidity, " +
+            "isAvailable=$isAvailable, " +
+            "accuracy=$accuracy)"
+    }
+
+    private fun getDewPointTemp(relativeHumidity: Float, actualTemp: Float): Double {
+        val h = ln(relativeHumidity / 100.0) + (17.62 * actualTemp) / (243.12 + actualTemp)
+        return 243.12 * (h / (17.62 - h))
+    }
```
Comment (shubhamsinghmutualmobile) [inline composesensors/src/main/java/com/mutualmobile/composesensors/RelativeHumiditySensorState.kt:70]: Do we need this anymore now that we have `dewPointTemperature` on line 41?

**Review by shubhamsinghmutualmobile — APPROVED** [review]

**Pull request closed without merging**